starter_code/.gitignore

```diff
@@ -1,2 +1,2 @@
-# Dependency directory
-node_modules
+node_modules,
```

Comment:
you don't need coma (,) 😉 that is why the node_modules keep uploading to github.
This has to be like this:
 `node_modules`
`.DS_Store`